pages/tools/cli.js

```diff
@@ -116,7 +125,15 @@ export default function CliPage() {
 										{
 											language: '.pkg',
 											code: getPkgCode()
-										}
+										},
+                    {
+                      language: 'windows',
+                      code: setUpWin()
+                    },
+                    {
+                      language: 'Linux',
```

Comment:
```suggestion
                      language: 'linux',
```

just so we are consistent across tabs

Reply:
That's great. Thank you @derberg. Is there anything I need to do?

Reply:
if you agree with change suggestion, you click `Commit suggestion` button